integrationTests/chainSimulator/staking/jail_test.go

```diff
@@ -0,0 +1,147 @@
+package staking
+
+import (
+	"encoding/hex"
+	"fmt"
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/multiversx/mx-chain-core-go/core"
+	"github.com/multiversx/mx-chain-core-go/data/transaction"
+	"github.com/multiversx/mx-chain-go/config"
+	"github.com/multiversx/mx-chain-go/node/chainSimulator"
+	"github.com/multiversx/mx-chain-go/node/chainSimulator/components/api"
+	"github.com/multiversx/mx-chain-go/node/chainSimulator/configs"
+	"github.com/multiversx/mx-chain-go/vm"
+	"github.com/stretchr/testify/require"
+)
+
+// Test scenario
+// 1. generate a new validator key
+// 2. do a stake transaction
+// 3. check validator is in waiting list and wait till validator is jailed
+// 4. do an unJail transaction
+// 5. staking v4 not enabled        --- node status should be new
```

Comment:
maybe collapse steps 5 & 6 and align them with the test runs?
Also, indenting went wrong on L27 & L28

Reply:
refactored